### PR TITLE
[KIECLOUD-322] - Support for EXTENSIONS_IMAGE and EXTENSIONS_IMAGE_NA…

### PR DESCRIPTION
**Pull request by swati-kale**
…MESPACE

Signed-off-by: Swati Kale <swkale@redhat.com>

### CODE REVIEW

File: frontend/src/component/operator-wizard/OperatorWizard.js [hunk 1]
```diff
@@ -350,7 +355,42 @@ export default class OperatorWizard extends Component {
...
+    const tempEnv = [
+      { name: EXTENSIONS_IMAGE_KEY, value: EXTENSIONS_IMAGE_VALUE },
+      {
+        name: EXTENSIONS_IMAGE_NAMESPACE_KEY,
+        value: EXTENSIONS_IMAGE_NAMESPACE_VALUE
+      }
+    ];
+
+    var server;
+    var obj = Dot.object(jsonObject);
```
Comment (yzhao583) [inline frontend/src/component/operator-wizard/OperatorWizard.js]: Minor: We may want to change "obj" to something meaningful?

File: frontend/src/component/operator-wizard/OperatorWizard.js [hunk 2]
```diff
@@ -350,7 +355,42 @@ export default class OperatorWizard extends Component {
...
+    if (
+      obj.spec !== undefined &&
+      obj.spec.objects !== undefined &&
+      obj.spec.objects.servers !== undefined
+    ) {
+      let servers = obj.spec.objects.servers;
+
+      for (server in servers) {
```
Comment (yzhao583) [inline frontend/src/component/operator-wizard/OperatorWizard.js]: We can say "let server in servers" and remove the "var server", and we may use map() here.

File: frontend/src/component/operator-wizard/OperatorWizard.js (hunk 1 above)
Comment (swati-kale) [reply frontend/src/component/operator-wizard/OperatorWizard.js]: updated

File: frontend/src/component/operator-wizard/OperatorWizard.js (hunk 2 above)
Comment (swati-kale) [reply frontend/src/component/operator-wizard/OperatorWizard.js]: updated

**Pull request merged**